### PR TITLE
Add ability to provide CA file via container environment

### PR DESCRIPTION
**Pull request by joshraker**
Also updated the test to ensure the MySQL server starts and SSL works.

### CODE REVIEW

**Comment by joshraker** [comment]
@almathew `SSL_CA_CERTIFICATE` is the env variable I chose. Let me know if you have another preference/suggestion.

**Comment by almathew** [comment]
I had this, but I can easily change: https://github.com/aptible/sweetness/pull/1208/files#diff-024e2a13ace88894ae192a2ace09b855e7bc400aaf1917728f940610c7fd99d6R120

I _might_ be able to test this out tonight with aptible-integration if you can push the images somewhere. Otherwise I probably won't get to it until tomorrow

**Comment by joshraker** [comment]
I updated this PR to match `CA_CERTIFICATE`. I'll work on publishing the images to my quay repo.

**Comment by joshraker** [comment]
Images have been pushed to this repo: https://quay.io/repository/josh_raker/mysql?tab=tags

File: bin/run-database.sh [hunk 1]
```diff
@@ -89,21 +89,23 @@ function mysql_initialize_certs () {
   mkdir -p "$SSL_DIRECTORY"
   pushd "$SSL_DIRECTORY"
 
+  local ca_cert_file="ca-cert.pem"
   local ssl_cert_file="server-cert.pem"
   local ssl_key_file="server-key.pem"
 
-  if [ -n "$SSL_CERTIFICATE" ] && [ -n "$SSL_KEY" ]; then
+  if [ -n "$CA_CERTIFICATE" ] && [ -n "$SSL_CERTIFICATE" ] && [ -n "$SSL_KEY" ]; then
```
Comment (almathew) [inline bin/run-database.sh:96]: Is there anyone using SSL cert/key alone? It sounds like that's not possible, right?

Comment (UserNotFound) [reply bin/run-database.sh:96]: It's not "possible", but there is one database configured this way. Before this PR, they would just not be able to connect with the `aptible` user at all. This change ensures this situation is prevented.

**Review by UserNotFound — APPROVED** [review]

**Pull request merged**